meson.build

```diff
@@ -539,6 +539,8 @@ features += {'vt.h': cc.has_header_symbol('sys/vt.h', 'VT_GETMODE')}
 
 features += {'consio.h': not features['vt.h'] and cc.has_header_symbol('sys/consio.h', 'VT_GETMODE')}
 
+features += {'wsdisplay_usl_io.h': cc.has_header_symbol('dev/wscons/wsdisplay_usl_io.h', 'VT_GETMODE', prefix: '#include <sys/types.h>')}
```

Comment:
Oh I should have asked before approving I guess but is the `prefix` part needed for this check?

Reply:
I don't like that it is `wsdisplay_usl_io.h`, we prefer hyphens for features name generally.

Reply:
> Oh I should have asked before approving I guess but is the `prefix` part needed for this check?

Yes, it is.